scripts/rstrnt-package

```diff
@@ -75,7 +75,7 @@ elif [ "$operation" = "remove" ]; then
     RC=$?
 elif [ "$operation" = "reinstall" ]; then
     _pkg_cmd $pkg_remove
-    _pkg_cmd $pkg_install
+    _pkg_cmd "$pkg_install"
```

Comment:
Should all the commands args be quoted?  ie _pkg_cmd "$pkg_remove"

Reply:
probably just fix up all the shell check concerns in there about a dozen.